modules/core/src/lib/deck-picker.js

```diff
@@ -56,6 +57,8 @@ export default class DeckPicker {
     if ('onError' in props) {
       this._onError = props.onError;
     }
+
+    Object.assign(this.props, props);
```

Comment:
Why keep a copy of every prop? This can potentially hold on to e.g. an outdated layer instance.

Reply:
Sure happy to change. My assumption was that these props are updated every frame, so holding on to things would be a non-issue, and conversely, why increase the bundle size with a bunch of code just to copy props?

Reply:
These are not updated every frame unless you are using React and stateless controller.

I don't see a bundle size issue with handling just one prop.